docs/api/native-theme.md

```diff
@@ -22,7 +22,38 @@ The `nativeTheme` module has the following properties:
 ### `nativeTheme.shouldUseDarkColors` _Readonly_
 
 A `Boolean` for if the OS / Chromium currently has a dark mode enabled or is
-being instructed to show a dark-style UI.
+being instructed to show a dark-style UI.  If you want to modify this value you
+should use `themeSource` below.
+
+### `nativeTheme.themeSource`
```

Comment:
. o O ( Naming thoughts )

Perhaps this should be `nativeTheme.overrideTheme` and the values should be `'light'`, `'dark'`, or `undefined`?

Reply:
@nornagon That's what is used to be but the underlying reason for this API is to back a 3-state switch and making a 3-state switch backed by a 2-state API (with the third state being "no value") felt worse than implementing the 3-state switch directly.